lms/djangoapps/mobile_api/testutils.py

```diff
@@ -202,6 +205,7 @@ def test_unenrolled_user(self):
         (None, False)
     )
     @ddt.unpack
+    @patch.dict(settings.FEATURES, {'ENABLE_MKTG_SITE': True})
     def test_visible_to_staff_only_course(self, role, should_succeed):
         self.init_course_access()
         self.course.visible_to_staff_only = True
```

Comment:
I see a lot of tests with `'ENABLE_MKTG_SITE'` set to `True`.  Are there any tests that ensure it does the right thing when the setting is `False`?